File: excercice1/main.java
class Playground {
    public static void main(String[ ] args) {
        Book newBook = new Book ("Naruto", 10);
        System.out.println(newBook.bookTitle + " " + newBook.bookPrice + "€");
    }
}



class Book {

public String bookTitle;
public int bookPrice;

public Book(String bookTitle, int bookPrice ) {

  this.bookTitle = bookTitle;
  this.bookPrice = bookPrice;
  }
}

File: excercice1/main.js
// Exercice 1 à réaliser en TypeScript, PHP et JavaCréez une classe pour représenter un livre. Cette classe doit contenir deux attributs:- un pour le titre du livre- un pour le prix du livreDans un algorithme, créez une instance de cette classe, et afficher le titre et le prix du livre.
var Book = /** @class */ (function () {
    function Book(name, price) {
        this.name = name;
        this.price = price;
    }
    return Book;
}());
;
var naruto = new Book('naruto', 10);
console.log(naruto);
